app/src/main/java/org/thoughtcrime/securesms/contacts/ContactAccessor.java

```diff
@@ -74,9 +74,9 @@ public List<String> getNumbersForThreadSearchFilter(Context context, String cons
     }
 
 //    if (context.getString(R.string.note_to_self).toLowerCase().contains(constraint.toLowerCase()) &&
-//        !numberList.contains(TextSecurePreferences.getLocalNumber(context)))
+//        !numberList.contains(MessagingModuleConfiguration.getShared().getPrefs().getLocalNumber(context)))
```

Comment:
If we're not using this code maybe remove?

Reply:
eek, find and replace fixed the commented code 🤔 

I'd be tempted to delete.